socorro/app/socorro_app.py

```diff
@@ -202,13 +203,18 @@ def _do_run(klass, config_path=None, values_source_list=None):
                 # pull in the application defaults from the 'application'
                 # configman option, once it has been defined
                 application_defaults_proxy,
+                # Get values from the app's config module
+                klass.config_module,
                 # pull in any configuration file
```

Comment:
Adding a common config for all apps would be as simple as adding a string to this list like `"socorro.config"` and putting the common config there.